HOWTO_RELEASE

```diff
@@ -0,0 +1,45 @@
+How to issue a GeoUtils release.
+
+ 1. Go to your local main repository (not the fork) and ensure your master branch is synced:
```

Comment:
> not the fork
Does this mean one need to clone the Glaciohack repo?

Reply:
yes.

Reply:
I recommend to have only one or two persons in charge of doing releases - this needs a bit coordination.